Base/Python/slicer/__init__.py

```diff
@@ -1,4 +1,108 @@
-""" This module sets up root logging and loads the Slicer library modules into its namespace."""
+""" This module sets up root logging and loads the Slicer library modules into its namespace.
+
+
+.. py:attribute:: app
+
+  This is set to the singleton instance of ``qSlicerApplication``::
+
+    >>> slicer.app
+    qSlicerApplication (qSlicerApplication at: 0x7ffd066a07a0)
+
+  .. warning::
+
+    This attribute is set only in the context of a running Slicer application.
+
+
+.. py:attribute:: mrmlScene
+
+  This is set to the instance of the MRML Scene::
+
+      >>> slicer.mrmlScene
+      <MRMLCore.vtkMRMLScene(0x2797cb0) at 0x7fbfedb82520>
+
+      >>> slicer.mrmlScene == slicer.app.mrmlScene()
+      True
+
+  .. warning::
+
+    This attribute is set only in the context of a running Slicer application.
```

Comment:
Instead of repeating this somewhat vague warninng several times, it could be better to create a subsection for all these attributes that are set only in the main Slicer GUI application.

Reply:
Ditto. Here is an issue tracking this:
* https://github.com/Slicer/Slicer/issues/7016

I the mean time, I will update the consistently update the waning based on your suggestion.